textile/chat-features.textile

```diff
@@ -923,14 +951,26 @@ h3(#chat-structs-presence-event). Presence Event
   }
 </pre>
 
-h3(#chat-structs-typing-event). Typing Event
+h3(#chat-structs-typing-event). Typing Event @(deprecated)@
 
 <pre>
   {
     "currentlyTyping": ["clientId-1", "clientID-2"],
   }
 </pre>
 
+h3(#chat-structs-typing-event-V2). Typing Event V2
+
+<pre>
+  {
+    "change": {
+        "event": "typing.started",
```

Comment:
Instead of `event` key, it is `type` in the DR => https://ably.atlassian.net/wiki/spaces/CHA/pages/3830513673/CHADR-093+Typing+Heartbeats+Ephemeral#Event-Changes
Which one should be preferred?

Reply:
maybe this needs to be updated